python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py

```diff
@@ -469,8 +469,8 @@ class Meta:
         description="Provides fields for testing behavior",
     )
 
-    autoMaterializeAssetEvaluations = graphene.Field(
-        non_null_list(GrapheneAutoMaterializeAssetEvaluationRecord),
+    autoMaterializeAssetEvaluationsOrError = graphene.Field(
+        non_null_list(GrapheneAutoMaterializeAssetEvaluationRecordsOrError),
```

Comment:
this shouldn't be a `non_null_list` anymore right?

Reply:
<img width="733" alt="Screen Shot 2023-05-30 at 2 05 28 PM" src="https://github.com/dagster-io/dagster/assets/2286579/2b71b27c-5c10-4146-96a7-5effc943a982">


this is the typescript type its outputting.